src/WebJobs.Script.WebHost/DependencyInjection/DependencyValidator/DependencyValidator.cs

```diff
@@ -48,7 +48,6 @@ private static ExpectedDependencyBuilder CreateExpectedDependencies()
                 .Expect<FileMonitoringService>()
                 .Expect<WorkerConsoleLogService>()
                 .Expect<FunctionInvocationDispatcherShutdownManager>()
-                .Expect<ChangeAnalysisService>()
```

Comment:
@fabiocav - Removed this for tests. Let me know if I should add unit tests for checking this list based on environment

Reply:
You can flag this as optional

Reply:
done! thanks